attending/vrn_map.js

```diff
@@ -34,9 +34,9 @@ function load_map() {
         [48.94, 7.24],
         [49.8, 10.22]
     ]);
-    L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-      attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, map tiles © <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA 2.0</a>',
-      maxZoom: 19
+    L.tileLayer('https://tileserver.memomaps.de/tilegen/{z}/{x}/{y}.png', {
```

Comment:
Please revert this change. ÖPNVKarte is not mentioned in the OSMF Privacy Policy.

Did you check that public transport mapping in OSM in Heidelberg is even useful at all? I mean the routes, not the stops.

Reply:
Yes, thanks for the hint. Didn't think about that.

Reply:
> Did you check that public transport mapping in OSM in Heidelberg is even useful at all? I mean the routes, not the stops.
[whb](https://www.openstreetmap.org/user/whb)  and I updated the area around the Hbf after finishing the tram & bus station. All tram and the most important bus lines should be correct again around the central station. The tiles öpnvkarte version should be fine now, routes and stations get updated until the SotM afaik: http://öpnvkarte.de/#8.6755;49.4039;17